.github/MAINTAINER.adoc

```diff
@@ -0,0 +1,79 @@
+= Maintainers
+ifdef::env-github[]
+:status:
+:outfilesuffix: .adoc
+:caution-caption: :fire:
+:important-caption: :exclamation:
+:note-caption: :paperclip:
+:tip-caption: :bulb:
+:warning-caption: :warning:
+endif::[]
+
+== Labels
+
+bug/possible:: (only for issues) if we need to analyze to understand if it's a bug or not. [bot only]
+bug/confirmed:: (only for issues) we are sure, it's a bug.
+bug/fix:: (only for PR) it's a bug fix.
+
+contributor/need-more-information:: we need more information from the contributor in order to analyze a problem.
+contributor/waiting-for-corrections:: (only for PR) we need the contributor to take actions in order to move forward with a PR
+contributor/needs-rebase:: (only for PR) use it only when there is some conflicts (and an automatic rebase is not possible) [bot, humans]
+
+duplicate:: it's a duplicate issue/PR.
+
+enhancement:: a new or improved feature
+
+question:: (only for issue) It's a question.
+
+proposal:: proposal PR/issues need a public debate
+* Proposal issues are design proposal that need to be refined with multiple contributors.
+* Proposal PRs are technical prototypes that need to be refined with multiple contributors.
+
+declined:: Rule #1 of open-source: no is temporary, yes is forever.
+
+WIP:: (only for PR) Work In Progress.
+
+platform/windows:: Windows related.
+
+area/provider/boltdb:: Boltd DB related.
```

Comment:
We also should have a `area/provider` one (something that would affect *all* the providers for example)